services/web/server/src/simcore_service_webserver/users_api.py

```diff
@@ -18,7 +18,7 @@
 from sqlalchemy import and_, literal_column
 
 from .db_models import GroupType, groups, tokens, user_to_groups, users
-from .groups_utils import convert_groups_db_to_schema
+from .groups.schemas import convert_groups_db_to_schema
```

Comment:
I see that you are importing it here. Why not put it in the `__init__.py` of groups.

Since it's a single utility function, maybe assign a better name to the module where it resides and import if from that module inside groups.